src/components/backend-ai-settings-view.ts

```diff
@@ -199,6 +199,9 @@ export default class BackendAiSettingsView extends BackendAIPage {
           --mdc-list-item__primary-text: {
             height: 20px;
           };
```

Comment:
Same here. Let's not add an extra style (`FixedMenuPosition` specific) if the component doesn't need it.